flutter_app/lib/command_constructor.dart

```diff
@@ -77,3 +77,33 @@ MavlinkFrame setMessageInterval(
   var frm = MavlinkFrame.v2(sequence, systemID, componentID, commandLong);
   return frm;
 }
+
+/// Constructs a MAVLink command to set the mode of the vehicle using MAV_CMD_DO_SET_MODE (176).
+///
+/// @sequence The sequence number for the MAVLink frame.
+/// Each component counts up its send sequence.
+/// Allows to detect packet loss.
+/// @systemId The MAVLink system ID of the vehicle (normally "1").
+/// @componentId The MAVLink component ID (normally "0").
+/// @customMode The new autopilot-specific mode. This is system-specific.
+/// @baseMode The new base mode.
+/// @customSubMode The new autopilot-specific sub-mode. This is system-specific.
+///
+/// @return A MAVLink frame representing the set mode command.
+MavlinkFrame setMode(int sequence, int systemID, int componentID, int customMode,
```

Comment:
I might be wrong but shouldn't baseMode come before customMode? From the Mavlink docs: https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_MODE